src/Libraries/CoreNodeModels/Properties/Resources.en-US.resx

```diff
@@ -494,10 +494,10 @@
     <comment>Description for Web Request</comment>
   </data>
   <data name="WebRequestPortDataResultToolTip" xml:space="preserve">
-    <value>The result of the web request.</value>
+    <value>Content of a webpage as a string.</value>
```

Comment:
```suggestion
    <value>Content of a web request as a string</value>
```

Reply:
@martinstacey not sure i would changes these files like this, you are basically just making a review right now, if you really want to do it like this, remember to `commit suggestion`. I do think that especially with these resource files its safer to change them from visual studio as it creates some files in the background when you do it from there.